### PR TITLE
chore: add Teddy Yongo

### PR DESCRIPTION
**Pull request by teyddie22**

### CODE REVIEW

**Review by Shivansh-yadav13 — APPROVED** [review]
Welcome!   :100: 🥳

File: public/data/teyddie22.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Teyddie",
```
Comment (Panquesito7) [inline public/data/teyddie22.json:2]:
```suggestion
  "name": "Teddy Yongo",
```

Comment (krishguptadev) [reply public/data/teyddie22.json:2]: Their name is their choice 😅

File: public/data/teyddie22.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing"
```
Comment (krishguptadev) [inline public/data/teyddie22.json:24]:
```suggestion
      "description": "Started freelancing!"
```

Comment (krishguptadev) [inline public/data/teyddie22.json:24]:
```suggestion
      "description": "Started freelancing!"
```

Comment (krishguptadev) [inline public/data/teyddie22.json:24]:
```suggestion
      "description": "Started freelancing!"
```

Comment (emmalearnscode) [reply public/data/teyddie22.json:24]: @krishguptadev exclamation marks should be their choice too 😉

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍

**Pull request merged**